### ISSUE TITLE
Package issue with AxiosError network error

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1933]
a customer reported in [slack](https://ibm-studios.slack.com/archives/C036P1CTN/p1718186830260419) an issue with NPM package as part of our Jest tests in a CIO Cirrus pipeline when using Node v20.9.0 and accessibility-checker v3.1.72 .  Downgrading to v3.1.71 resolves this issue.

AxiosError: network error

<img width="1528" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNTlnQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--034d0bc29b37d8d58b42e5f290338f5b8eed0989/image.png" alt="image.png" />

**Comment by tombrunet** [comment]
User confirmed working fine for 3.1.73. Must have been some temporary issue or transient deploy failure. But, working now.
